src/osd/OSD.cc

```diff
@@ -7077,7 +7085,13 @@ void OSD::ms_fast_dispatch(Message *m)
     tracepoint(osd, ms_fast_dispatch, reqid.name._type,
         reqid.name._num, reqid.tid, reqid.inc);
   }
-
+#ifdef HAVE_JAEGER
+  op->set_osd_parent_span(dispatch_span);
+  if(op->osd_parent_span){
```

Comment:
nit: all the ifs should be followed by a space, i.e. 
```suggestion
  if (op->osd_parent_span){
```